packages/components/src/DataList/components/DataListHeaderTile/DataListHeaderTile.css

```diff
@@ -1,5 +1,6 @@
 .headerLabel {
   display: flex;
+  position: relative;
```

Comment:
This is keeping the sorting dropdown in the right spot. Otherwise you get this:

![Screenshot 2024-01-17 at 5 17 22 PM](https://github.com/GetJobber/atlantis/assets/104797704/96cf14c9-1dbe-4f12-b7aa-dcb6df45764a)